src/plugins/lua/known_senders.lua

```diff
@@ -251,8 +242,8 @@ local function check_known_incoming_mail_callback(task)
     if list_of_senders then
       for _, sndr in ipairs(list_of_senders) do
         if sndr == sender then
-          task:insert_result('INC_MAIL_KNOWN', 1.0,
-                  string.format('Incoming mail and it\'s sender is known'))
+          task:insert_result(settings.symbol_check_mail, 1.0,
+                  string.format('Incoming mail with sender %s is known', sndr))
```

Comment:
`          task:insert_result(settings.symbol_check_mail, 1.0, sndr)`

Symbol options should be kept short where possible.